frontend/src/actions/resources.js

```diff
@@ -168,8 +172,9 @@ const informationForUSCitizens = withStory((story) => {
 const informationForUSTravelers = withStory((story) => {
   const usCitizenAbroad = {
     predicate:
-      story.countryOfOrigin === countries.USA &&
-      ![countries.USA, countries.MEX].includes(story.country),
+      (story.countryOfOrigin === countries.USA ||
+        story.country === countries.US) &&
+      ![countries.USA, countries.MEX, countries.US].includes(story.country),
     link:
```

Comment:
`(story.countryOfOrigin === countries.USA || story.country === countries.US) ` should be `(story.countryOfOrigin === countries.USA || story.countryOfOrigin === countries.US) `